android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -322,6 +322,7 @@ constructor(
         composition
           .retrieveCompositionSections()
           .groupBy { it.focus.reference?.split(TYPE_REFERENCE_DELIMITER)?.firstOrNull() ?: "" }
+          .asSequence()
```

Comment:
This can be moved before groupBy

Reply:
@ellykits Adding `asSequence()` before `groupBy` is redudant. You have a look.